lua/entities/wtib_harvester_large/init.lua

```diff
@@ -9,6 +9,7 @@ util.PrecacheSound("apc_engine_stop")
 
 ENT.MaxDrain = 400
 ENT.Range = 400
+ENT.Focus = 80
```

Comment:
it is used in the red beam that i added that is fired from the harvester when its harvesting. Even though it says its for debug, it though it looked cool, like the harvester was firing a laser at it.